src/main/java/com/adobe/commerce/cif/model/product/AbstractProduct.java

```diff
@@ -28,16 +28,16 @@
     @ApiModelProperty(value = "The internal unique ID of the product in the commerce backend system.", required = true)
     protected String id;
 
-    @ApiModelProperty(value = "The unique SKU of the product assigned by the vendor or manufacturer.")
+    @ApiModelProperty(value = "The unique SKU of the product assigned by the vendor or manufacturer.", required = true)
```

Comment:
This cannot be mandatory, because some shop systems like Commercetools don't have a sku for the base/parent product. Or we enforce that a sku (like the sku of the first variant) should be set?

Reply:
Product variants however should always have a sku, so it is required. So I tend to go for a workaround sku for base products.

Reply:
Or we should not define a `sku` in the abstraction and correctly handle it in the implementation.

Reply:
You mean moving the sku from the AbstractProduct to both the Product (as non required) and the ProductVariant (required) classes? Sounds good to me.

Reply:
I think we added `sku` on product level by intention to be more flexible in the product data design we support. @cjelger remember?

Reply:
Well initially we didn't have a `sku` on the base product, so there was no question where to put it. ;-) I'm fine with Mark's proposal.

Reply:
Ok, also +1 for Mark's proposal then.

Reply:
Done.